src/lib/swapper/swappers/utils/helpers/helpers.test.ts

```diff
@@ -34,3 +41,26 @@ describe('normalizeIntegerAmount', () => {
     expect(result4).toEqual('586084736227728')
   })
 })
+
+describe('getTreasuryAddressForReceiveAsset', () => {
+  it('gets the treasury address for an ERC20 asset', () => {
+    const treasuryAddress = getTreasuryAddressForReceiveAsset(foxAssetId)
+    expect(treasuryAddress).toStrictEqual(DAO_TREASURY_ETHEREUM_MAINNET)
+  })
+
+  it('gets the treasury address for ETH asset', () => {
+    const treasuryAddress = getTreasuryAddressForReceiveAsset(ethAssetId)
+    expect(treasuryAddress).toStrictEqual(DAO_TREASURY_ETHEREUM_MAINNET)
+  })
+
+  it('gets the treasury address for Optimism asset', () => {
+    const treasuryAddress = getTreasuryAddressForReceiveAsset(optimismAssetId)
+    expect(treasuryAddress).toStrictEqual(DAO_TREASURY_OPTIMISM)
+  })
+
+  it('throws for unsupported chains', () => {
```

Comment:
The `chains` terminology in this test naming confirms the intent the ChainId vernacular would make more obvious

Reply:
https://github.com/shapeshift/web/commit/0348c29fd253d00f8152957a948d872a9941a8cf